### DISCUSSION TITLE
Did something wrong with ESP32S3. Now bricked ?

### DISCUSSION THREAD

**Discussion by gert4git** [espressif/arduino-esp32#7272] [General]
Hello All,

New user of ESP32S3 here. Using 'LilyGo T-Display S3' with Arduino IDE.
https://github.com/Xinyuan-LilyGO/T-Display-S3

I got example code upload and working yesterday. All was good and I tried some TFT code.
Today I wanted to use Arduino IDE serial port monitor and tried following instructions from here : 
 https://github.com/espressif/arduino-esp32/discussions/6865
But I did something wrong and it did not work. 

Now even going back to same code that was OK yesterday it fails to upload today.
I or the serial port experiments must have done something bad.
Every upload today is failing. The device looks like a brick.

Here is a screen copy:
https://www.skywatcher.space/download/TDispS3_Uploadfail.jpg

I know there is a ton of troubleshooting for people seeing the 'fail to connect' error message and it refers to initial setup of the connection being faulty. But it worked yesterday. I need to get back to that config.

Would you be able to help?

Thanks,
Gert

**Comment by me-no-dev** [comment]
Try holding the boot button while it's trying to flash

**Reply by gert4git** [reply]
Ah, OK. That did reset the board and it now takes uploads again. That's a scary failure mode. should be more clearly documented how to recover from it.
Cheers,
Gert

**Reply by me-no-dev** [reply]
Don't be scared. You can always manually boot into download mode by resetting while holding the boot button. That is well known and documented :)

**Reply by robsannaa** [reply]
Did your device showed up in tue device manager?

**Reply by AlHeeley** [reply]
Mines bricked, it doesnt show up at all in device manager, so cannot assign any com port to it. THe manual booting doesnt work either.
